README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+# 1. Поддерживаемые версии Greenplum
+
+На данный момент поддерживается только Greenplum 6.
+
+# 2. Компиляция pgBackRest
+
+- Установить зависимости
+
+для CentOS 7:
+```
+yum install gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel lz4-devel libyaml-devel
+```
+
+для ALT Linux p10
+```
+apt-get update
+apt-get install git gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel libyaml-devel zlib-devel 
+```
+
+- Добавить в переменную окружения PATH путь к pg_config Greenplum
+
+- Скачать репозиторий
+```
+git clone https://github.com/arenadata/pgbackrest -b 2.50-ci
+```
+
+- Перейти в каталог с исходным кодом
+```
+cd pgbackrest/src/
+```
+
+- Запустить скрипт конфигурирования
+
+Без параметров будут включены оптимизация (-O2) и векторизация вычисления контрольных сумм страниц.
+```
+./configure
+```
+
+Конфигурирование для отладки рекомендуется выполнять командой
+```
+CFLAGS="-O0 -g3" ./configure --disable-optimize --enable-test
+```
+
+- Собрать
+
+Команда для сборки с использованием всех доступных ядер и без отображения исполняемых команд
+```
+make -j`nproc` -s
+```
+
+В результате в текущем каталоге появится исполняемый файл с именем `pgbackrest`.
+
+- Установить
+
+```
+sudo make install
+```
+По умолчанию исполняемый файл `pgbackrest` будет размещен в каталог `/usr/local/bin`. Это единственное действие, которое выполняется на этапе установки.
+
+
+# 3. Настройка
+- Создать каталоги для размещения бэкапа и логов. Пусть для теста это будут `/tmp/backup` и `/tmp/backup/log`.
+```
+mkdir -p /tmp/backup/log
+```
+
+- Создать конфигурационный файл
+
+Для **стандартного демонстрационного кластера**, созданного с `DATADIRS=/tmp/gpdb`, этот файл будет выглядеть так:
+```
+[seg-1]
+pg1-path=/tmp/gpdb/qddir/demoDataDir-1
+pg1-port=6000
+
+[seg0]
+pg1-path=/tmp/gpdb/dbfast1/demoDataDir0
+pg1-port=6002
+
+[seg1]
+pg1-path=/tmp/gpdb/dbfast2/demoDataDir1
+pg1-port=6003
+
+[seg2]
+pg1-path=/tmp/gpdb/dbfast3/demoDataDir2
+pg1-port=6004
+
+[global]
+repo1-path=/tmp/backup
+log-path=/tmp/backup/log
+start-fast=y
+fork=GPDB
+```
+
+В дальнейших примерах команд предполагается, что конфигурационный файл имеет стандартное имя - `/etc/pgbackrest.conf`. Если требуется использовать другой файл, то его имя можно передать через параметр `--config`.
```

Comment:
I think it is better to place this chapter above the chapter with the description of the config file contents.
Thus it will be more logical - first we create a file, then we fill it,